src/scenes/game-scene.ts

```diff
@@ -97,7 +97,7 @@ class GameScene extends Phaser.Scene {
     this.level = new Level(this, levelConfig);
     this.audio = new Audio(this, soundConfig);
 
-    this.audio?.playAudio('music2');
+    this.audio.playAudio('music2');
```

Comment:
Does this not complain about this.audio potentially being undefined?

Reply:
`this.audio` defined in the line above this one!
so no, it doesn't complain...